### DISCUSSION TITLE
Cloudflare Tunnel error

### DISCUSSION THREAD

**Discussion by Starc73** [indomus/forum#3191] [HUB: Home Assistant]
ragazzi chiedo gentilmente il vs. aiuto ho installato Home Assistant OS su un nuc ed ho eseguito la guida per il collegamento remoto come da guida https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/ e alla fine della procedura ho il seguente errore:
"Please open the following URL and log in with your Cloudflare account:

https://dash.cloudflare.com/argotunnel?aud=&callback=https%3A%2F%2Flogin.cloudflareaccess.org%xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
Leave cloudflared running to download the cert automatically.
Failed to write the certificate due to the following error:
Get "https://login.cloudflareaccess.org/jQAo0SNefm1Gxxxxxxxxxxxxxxx=": dial tcp 198.41.214.156:443: connect: no route to host

Your browser will download the certificate instead. You will have to manually
copy it to the following path:

/root/.cloudflared/cert.pem
Get "https://login.cloudflareaccess.org/jQAo0SNefm1GwLpSMY-RKcA9q3RhY8ZQDgcphFBwJHw=": dial tcp 198.41.214.156:443: connect: no route to host
s6-rc: warning: unable to start service init-cloudflared-config: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container."
cosa devo fare ?

grazie a tutti

**Comment by AleDestroyer** [comment]
Ciao,
dovresti collegarti al link che ti viene propesto, loggarti con le credenziali cloudflare ed autorizzare la zona (dominio) al tunnel.

**Reply by Starc73** [reply]
Ho dato l'autorizzazione al tunnel su cloudflare ma poi mi presenta il messaggio che vedi. C'è un modo di verificare su cloudflare se il tunnel è attivo ? Hai altri consigli da darmi ?

grazie

**Reply by AleDestroyer** [reply]
Accedi a cloudflare, poi nel menù a sinistra vai su
 → zero trust → networks → tunnels

**Comment by Starc73** [comment]
Ho provato a rifare tutto con una mail diversa in cloudflare nuovi dns, ecc... ed ecco il solito messaggio
"
2024-08-21T20:55:02Z INF Waiting for login...
2024-08-21T20:55:55Z INF Waiting for login...
2024-08-21T20:56:48Z INF Waiting for login...
2024-08-21T20:57:41Z INF Waiting for login...
2024-08-21T20:58:34Z INF Waiting for login...
2024-08-21T20:59:27Z INF Waiting for login...
2024-08-21T21:00:20Z INF Waiting for login...
2024-08-21T21:01:13Z INF Waiting for login...
2024-08-21T21:02:06Z INF Waiting for login...
Failed to write the certificate due to the following error:
Failed to fetch resource

Your browser will download the certificate instead. You will have to manually
copy it to the following path:

/root/.cloudflared/cert.pem
Failed to fetch resource
s6-rc: warning: unable to start service init-cloudflared-config: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container."
scusate mi potreste dare come scaricare sto certificato manualmente ?

grazie
